graftm/run.py

```diff
@@ -381,7 +381,8 @@ def graft(self):
                                                         filter_minimum
                                                         )
                     if aln_result:
-                        seqs_list.append(hit_aligned_reads)
+                        setattr(unpack, 'alignment_path', hit_aligned_reads)
```

Comment:
I think adding an `alignment_path` instance variable to an unpack object which ostensibly an object for representing where a read comes from is making things a bit too confusing, yeh?

The problem is some information flow? I'm a bit lost so this could be off base, but does it make sense to invent a new class that holds an alignment path and whether they have slash endings, or something along those lines?